### PR TITLE
Notes and Recommendations from Ken

### PR DESCRIPTION
**Pull request by kenleezle**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -85,6 +85,7 @@ app.use(bodyParser.urlencoded({
   extended: false
 }));
 
+// flag
 var User = connection.define('user', {
```
Comment (kenleezle) [inline server.js]: User and Review should each be moved into their own file, in a directory called `models`.
then in this file you should do:
```
var User = require('models/user.js');
```

File: server.js [hunk 2]
```diff
@@ -147,6 +148,7 @@ var Restaurant = sequelize.define('Restaurants', {
 });
 */
 
+// flag
```
Comment (kenleezle) [inline server.js]: ditto here

File: server.js [hunk 3]
```diff
@@ -280,6 +282,7 @@ app.post("/saveRating", function(req, res) {
   });
 });
 
+//flag
```
Comment (kenleezle) [inline server.js]: restful routes follow a convention. these routes do not. here is the convention documented which you should follow:

http://guides.rubyonrails.org/routing.html#crud-verbs-and-actions

that is not just the convention for rails, it is for REST, its just that rails documentation is good so...

Comment (kenleezle) [inline server.js]: also note how:

`/edit/:id` and `/updateReview/:id` are not following the same convention.

they should be:
* `GET /reviews/edit/:id`
* `POST/PATCH/PUT /reviews/edit/:id`

**Comment by MrMendonez** [comment]
Great advice. Thank you.

**Pull request closed without merging**